cmd/warmer/cmd/root.go

```diff
@@ -87,6 +95,8 @@ func addKanikoOptionsFlags() {
 	RootCmd.PersistentFlags().VarP(&opts.RegistriesCertificates, "registry-certificate", "", "Use the provided certificate for TLS communication with the given registry. Expected format is 'my.registry.url=/path/to/the/server/certificate'.")
 	RootCmd.PersistentFlags().VarP(&opts.RegistryMirrors, "registry-mirror", "", "Registry mirror to use as pull-through cache instead of docker.io. Set it repeatedly for multiple mirrors.")
 	RootCmd.PersistentFlags().StringVarP(&opts.CustomPlatform, "customPlatform", "", "", "Specify the build platform if different from the current host")
+	RootCmd.PersistentFlags().StringVarP(&opts.DockerfilePath, "dockerfile", "d", "Dockerfile", "Path to the dockerfile to be built.")
```

Comment:
nit: might make sense to add some more information in the help text here, this was my attempt:

s/Path to the dockerfile to be built./Path to a dockerfile that the kaniko warmer will write out each stage's base image layers from.   Using the same dockerfile as what you plan to build in the kaniko executor will "warm" the cache for that build.